osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerUserModDisplay.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Extensions.ObjectExtensions;
+using osu.Game.Online.Multiplayer;
+using osu.Game.Online.Rooms;
+using osu.Game.Rulesets;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Screens.Play.HUD;
+
+namespace osu.Game.Screens.OnlinePlay.Multiplayer
+{
+    public partial class MultiplayerUserModDisplay : ModDisplay
+    {
+        [Resolved]
+        private MultiplayerClient client { get; set; } = null!;
+
+        [Resolved]
+        private RulesetStore rulesets { get; set; } = null!;
+
+        public new Bindable<IReadOnlyList<Mod>> Current
+        {
+            get => throw new NotSupportedException("Cannot access the current mods of a multiplayer mods display.");
+            set => throw new NotSupportedException("Cannot access the current mods of a multiplayer mods display.");
+        }
```

Comment:
- does this even matter?
- at this point make this a `CompositeDrawable` with a `ModDisplay` nested rather than subclass or something